integrations/kafka/documentation.yaml

```diff
@@ -0,0 +1,96 @@
+exporter_type: generic_standalone
```

Comment:
What is "generic_standalone" supposed to convey? The only values we currently support are "standalone", "sidecar", "included", and "infrastructure"? Does this need to be handled specially, or was it intended to be "standalone"?

Reply:
Standalone is correct, our mistake.